inscriptions/ISic000524.xml

```diff
@@ -190,14 +196,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#GC">
+                <ab>
+                    <lb n="1"/><gap reason="lost" quantity="4" unit="character"/>rus
+                    <lb n="2"/><name>Flaviae</name>
+                    <lb n="3"/>benemer<gap reason="lost" quantity="3" unit="character"/>
```

Comment:
benemer is presumably for bene merenti, and I think could therefore be given as bene mer[enti]